### DISCUSSION TITLE
Detect Version

### DISCUSSION THREAD

**Discussion by sschmocker** [Romanitho/Winget-Install#26] [Q&A]
Hi
I figured it would be nice to have the option to set a flag in the detectionscript that, if checked, also checks the repo if the latest version is already installed. 
This way we could control updates via the detectionscript itself. 
Since we dont use your winget upgrade script this would be a nice to have.

**Comment by Romanitho** [comment]
Hi,
In that case, I would suggest two things:
- use WAU (our winget upgrade script) with the white list option. In that case, it will maintain only apps you want.
- or, use the work of @o-l-a-v that is exactly what you want (if I don't say wrong)
